lib/extensions/ar_migration.rb

```diff
@@ -52,11 +28,41 @@ def self.update_local_migrations_ran(version, direction)
         WHERE id = #{region.id}
       SQL
     end
+
+    class RemoteRegionThing
+      attr_reader :region, :subscription, :version
+      def initialize(subscription, version)
+        @region       = MiqRegion.find_by(:region => subscription.provider_region)
```

Comment:
Avoid using model in migration (same reasons for why we don't allow them in manageiq-schema)

Reply:
I'm not sure what the alternative would be here. Should we do something like the stubs we use for migrations? We're not using any model methods, all we need it for is to query the `migrations_ran` attribute.

Reply:
Something like this maybe?
```ruby
module ArPglogicalMigration
  class NotMiqRegion
    self.table_name = "miq_regions"
ennd
```